pkg/types/envd.go

```diff
@@ -46,6 +46,24 @@ type EnvdInfo struct {
 	types.Info
 }
 
+type EnvdContext struct {
+	Current  string    `json:"current,omitempty"`
+	Contexts []Context `json:"contexts,omitempty"`
+}
+
+type Context struct {
+	Name          string      `json:"name,omitempty"`
+	Builder       BuilderType `json:"builder,omitempty"`
+	BuilderSocket string      `json:"builder_socket,omitempty"`
+}
+
+type BuilderType string
+
+const (
+	BuilderTypeDocker     BuilderType = "docker-container"
+	BuilderTypeKubernetes BuilderType = "kube-pod"
```

Comment:
Is this supported in this PR?

Reply:
No, it will be supported in the next PR.